### ISSUE TITLE
pritunl v1.30.3157.70 

### ISSUE THREAD

**Issue by captainhook** [simonmicro/Pritunl-Fake-API#21]
Looks like the css must've changed?

**Comment by captainhook** [comment]
I reverted to 1.30.3116.68 and the css isn't loading. Anyone else seeing the same?
I was testing some of the advanced features... should've kept a backup. I wish they had >7day trials.

**Comment by simonmicro** [comment]
Hmm, could you send a screenshot of what is missing? Also, did you try to clean your browser cache (just-in-case)?

**Comment by captainhook** [comment]
Well, everything looks missing... it looks like no css is being applied. I have cleared browser cache, rebooted the vm.

I did attempt to host on my own server and logged the response by changing line 129 in index.php to 'true', and the response looks fine (https://pastebin.com/Q2mvYbWU). I've checked using Firefox DevTools and the extra 'enterprise' css file isn't there, the rest of the inline css remains unchanged. Very odd.
